docs/rfcs/0010-custom-stage.md

```diff
@@ -0,0 +1,255 @@
+- Start Date: 2023-02-20
+- Target Version: 0.41.4
+
+# Summary
+
+This RFC introduces a new way to enable users to use “custom stages” that users defined in their pipelines.
+
+# Motivation
+
+Currently, users can use only stages that PipeCD have already defined. But some users want to define new stages by their use-cases as bellow. 
+
+- Deploying infrastructure by tools other than that PipeCD supports (terraform and kubernetes) such as SAM, cloud formation….
+- Building Docker images with Kaniko
+- Running integration tests
```

Comment:
These are the role of CD? 
I think such a function could be actualized, but I feel it's not appropriate mentioning it here.